src/util/FileUtils.cpp

```diff
@@ -41,8 +41,7 @@
 
 #include <pdal/util/FileUtils.hpp>
 #include <pdal/util/Utils.hpp>
-
-using namespace std;
+#include <pdal/pdal_types.hpp>
```

Comment:
Not that it makes  any difference, but I don't think "using namespace std" hurts anything here and just means you have std:: all over the place.